Libraries/LibWeb/HTML/HTMLElement.cpp

```diff
@@ -460,15 +460,10 @@ int HTMLElement::offset_top() const
     // NOTE: Ensure that layout is up-to-date before looking at metrics.
     const_cast<DOM::Document&>(document()).update_layout();
 
-    if (!layout_node())
+    if (!paintable_box())
```

Comment:
Would null-checking `paintable()` not be the less intrusive change, and would still allow inline elements e.g `<span>` to have non-zero `offsetLeft` etc?

Reply:
Right, that was actually my original change, but then I noticed the other functions in this class used paintable_box, and from what I understand those should be consistent (or is this different for a reason I am missing)

Based on that though, maybe I should find/write myself a testcase which covers both those cases 🤔

Reply:
Oh, I completely missed a FIXME in those functions, yeah, I should have stuck just using paintable. Will try find or write a test for whatever this change as-is would be breaking